src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java

```diff
@@ -75,16 +75,24 @@ public User createUser(User newUser) {
         return newUser;
     }
 
-    public User updateUser(Long id, UserUpdateDTO userUpdateDTO, String updatingUsername) {
+    public User updateUser(Long id, User userInput, String updatingUsername) {
```

Comment:
A DTO should be used here for sure, in case we want to add new properties to the user